plugins/modules/azure_rm_monitoractivitylogalerts.py

```diff
@@ -0,0 +1,520 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2025 Klaas Demter (@Klaas-)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: azure_rm_monitoractivitylogalerts
+version_added: "3.6.0"
```

Comment:
The next version should be v3.5.0. Thank you!
```suggestion
version_added: "3.5.0"
```

Reply:
my understanding is that 3.5.0 is released, https://github.com/ansible-collections/azure/releases/tag/v3.5.0 -- so it should be 3.6.0 ? :)